test/server/cards/01_SOR/events/ForceThrow.spec.ts

```diff
@@ -0,0 +1,113 @@
+describe('Force Throw', function() {
+    integration(function(contextRef) {
+        describe('Force Throw\'s ability', function() {
+            it('should select opponent to discard a card and deal damage to a unit equals to its cost', function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['force-throw'],
+                        groundArena: ['wampa', 'ezra-bridger#resourceful-troublemaker']
+                    },
+                    player2: {
+                        hand: ['karabast', 'battlefield-marine'],
+                        groundArena: ['specforce-soldier', 'atst'],
+                        spaceArena: ['tieln-fighter']
+                    }
+                });
+
+                const { context } = contextRef;
+
+                // Damage dealt by opponent discarded card
+                context.player1.clickCard(context.forceThrow);
+                expect(context.player1).toHaveExactPromptButtons(['You', 'Opponent']);
+
+                // Opponent discards a card
+                context.player1.clickPrompt('Opponent');
+                context.player2.clickCard(context.karabast);
+
+                // Validate board state and options
+                expect(context.karabast).toBeInZone('discard');
+                expect(context.player1).toHaveEnabledPromptButton('Pass');
```

Comment:
We should use `expect(context.player1).toHavePassAbilityPrompt('Deal damage to a unit equal to the cost of Karabast (2 damage)');`